powerauth-webauth-client/src/main/java/io/getlime/security/powerauth/app/webauth/demo/configuration/WebAuthServiceConfiguration.java

```diff
@@ -73,7 +86,45 @@ public String getClientSecret() {
      */
     @Bean
     public NextStepClient defaultNextStepClient() {
-        return new NextStepClient(nextstepServiceUrl);
+        NextStepClient client = new NextStepClient(nextstepServiceUrl);
+        // whether invalid SSL certificates should be accepted
+        if (acceptInvalidSslCertificateInNextStep) {
+            trustAllCertificates();
+        }
+        return client;
+    }
+
+    /**
+     * Activate trust in all SSL certificates including invalid ones for non-production use.
+     */
+    private void trustAllCertificates() {
+        HttpsURLConnection.setDefaultHostnameVerifier((hostname, session) -> true);
```

Comment:
Same issue as above - this setting works for all `HttpsURLConnection` instances, there is no benefit in having two separate attributes for it and current behavior is surprising ("I only disabled SSL on Next Step, and it's disabled on Credential Store as well), hence incorrect.